### PR TITLE
add README files for Microsoft.Win32.*

### PR DESCRIPTION
**Pull request by smasher164**

### CODE REVIEW

File: src/libraries/Microsoft.Win32.Primitives/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
+
+## Deployment
+
+`Microsoft.Win32.Primitives` is provided as a [NuGet package](https://www.nuget.org/packages/Microsoft.Win32.Primitives/).
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Primitives/README.md:15]: Incomplete. This is also offered as an inbox package in Microsoft.NETCore.App:
 https://github.com/dotnet/runtime/blob/493574ee470daf8a353203ce7e933f931f986159/src/libraries/NetCoreAppLibrary.props#L33

See this comment, needs to look like this: https://github.com/dotnet/runtime/pull/78121/files#r1018292210

File: src/libraries/Microsoft.Win32.Registry/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
+
+## Deployment
+
+`Microsoft.Win32.Registry` is provided as a [NuGet package](https://www.nuget.org/packages/Microsoft.Win32.Registry/).
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Registry/README.md:15]: Also incomplete: https://github.com/dotnet/runtime/blob/493574ee470daf8a353203ce7e933f931f986159/src/libraries/NetCoreAppLibrary.props#L34

Needs to be like https://github.com/dotnet/runtime/pull/78121/files#r1018292210

File: src/libraries/Microsoft.Win32.Registry.AccessControl/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+This assembly provides APIs for managing access control lists for Microsoft.Win32.RegistryKey.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Registry.AccessControl/README.md:11]: This is missing the "help wanted" label in the query.

```suggestion
See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32+label%3A%22help+wanted%22) issues.
```

File: src/libraries/Microsoft.Win32.Registry/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+This assembly provides APIs for accessing and modifying the Windows Registry.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Registry/README.md:11]: Same

```suggestion
See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32+label%3A%22help+wanted%22) issues.
```

File: src/libraries/Microsoft.Win32.SystemEvents/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+This assembly provides support for accessing Windows system event notifications.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.SystemEvents/README.md:11]:
```suggestion
See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32+label%3A%22help+wanted%22) issues.
```

File: src/libraries/Microsoft.Win32.Registry.AccessControl/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Registry.AccessControl
+
+This assembly provides APIs for managing access control lists for Microsoft.Win32.RegistryKey.
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Registry.AccessControl/README.md:3]: We should match the message we have in the package: https://github.com/dotnet/runtime/blob/28cf1719704d33a677990cf487af5719b6389f2a/src/libraries/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj#L6

It is also the message we publish in the NuGet Readme: https://www.nuget.org/packages/Microsoft.Win32.Registry.AccessControl#readme-body-tab

```suggestion
This assembly provides APIs for managing access and audit control lists for Microsoft.Win32.RegistryKey.
```

File: src/libraries/Microsoft.Win32.Registry/README.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Registry
+
+This assembly provides APIs for accessing and modifying the Windows Registry.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
```
Comment (carlossanlop) [inline src/libraries/Microsoft.Win32.Registry/README.md:9]: I am not confident we want this contribution bar for this one. The last version where we published it was in 5.0: https://www.nuget.org/packages/Microsoft.Win32.Registry 

@danmoseley what would you say should be our bar for an assembly like this?

Comment (smasher164) [reply src/libraries/Microsoft.Win32.Registry/README.md:9]: Is the issue with the contribution bar about accepting refactoring changes?

Comment (smasher164) [reply src/libraries/Microsoft.Win32.Registry/README.md:9]: Bump. Do we want to accept refactoring changes for this package? Is this something we can fix later on?
/cc @carlossanlop @danmoseley

Comment (carlossanlop) [reply src/libraries/Microsoft.Win32.Registry/README.md:9]: @ViktorHofer @ericstj would you agree that this is the right bar for this assembly?

Comment (ViktorHofer) [reply src/libraries/Microsoft.Win32.Registry/README.md:9]: We just recently accepted a refactoring change in Registry, so I would say yes, we do.

**Review by carlossanlop — APPROVED** [review]
:shipit:

File: src/libraries/Microsoft.Win32.Registry/README.md (hunk 7 above)
Comment (teo-tsirpanis) [reply src/libraries/Microsoft.Win32.Registry/README.md:9]: > The last version where we published it was in 5.0: https://www.nuget.org/packages/Microsoft.Win32.Registry

@carlossanlop it became inbox in 6.0.

**Pull request merged**